ts/session/apis/snode_api/onions.ts

```diff
@@ -567,6 +567,24 @@ async function processOnionResponse({
   }
 }
 
+async function processNoSymmetricKeyError(
+  guardNode: Snode,
+  symmetricKey?: ArrayBuffer
+): Promise<ArrayBuffer> {
+  if (!symmetricKey) {
+    window?.log?.error(
```

Comment:
Could we store the error message as a const and then log it and throw the error?

Reply:
Updated